Fix range filter bug on paging, don't sort on values

lbrant1 opened the pull request:
Only ignore a key if all it's data is empty. In the case of range
filter, we need to know which value we are filtering on (the greater
than or less than) and it is fine if one of those values is Falsy

Commit:
Fix range filter bug on paging, don't sort on values

Only ignore a key if all it's data is empty. In the case of range
filter, we need to know which value we are filtering on (the greater
than or less than) and it is fine if one of those values is Falsy

lefcourn commented on seeker/views.py:
```diff
@@ -315,16 +315,14 @@ def normalized_querystring(self, qs=None, ignore=None):
-                values = sorted(values)
             parts.extend(urlencode({key: val}) for val in values)
         return '&'.join(parts)
+    pass
```
What is this here for?

lbrant1 committed:
Removed accidentally added pass line

lefcourn commented on seeker/views.py:
```diff
@@ -315,14 +315,11 @@ def normalized_querystring(self, qs=None, ignore=None):
             if ignore and key in ignore:
                 continue
-            if not data[key]:
+            if not data.getlist(key) or not any(data.getlist(key)):
```
Can we put "values = data.getlist(key)" above this and just do the "any" check?

lbrant1 committed:
Set values before check, make sure contains at least one true item in list

Since all items passed in are string values, the only false items would be an empty values list or a a list of empty strings.

dcwatson commented:
Did you mean for this to go into master, or 4.0?
